diff --git a/Inj/Base.hs b/Inj/Base.hs
index 19ee50f..c268b21 100644
--- a/Inj/Base.hs
+++ b/Inj/Base.hs
@@ -128,16 +128,6 @@ class d ~ DecideZipList p => InjZipList d p a where
 instance InjZipList (DecideZipList p) p a => Inj p (ZipList a) where
   inj = injZipList
 
-type family DecideOption p where
-  DecideOption (Option p) = Decision_Map
-  DecideOption p = Decision_Wrap
-
-class d ~ DecideOption p => InjOption d p a where
-  injOption :: p -> Option a
-
-instance InjOption (DecideOption p) p a => Inj p (Option a) where
-  inj = injOption
-
 type family DecideST p where
   DecideST (ST s p) = Decision_Map
   DecideST p = Decision_Wrap
@@ -323,12 +313,6 @@ instance
   where
     injZipList = pure . inj
 
-instance
-    (DecideOption p ~ Decision_Wrap, Inj p a) =>
-    InjOption Decision_Wrap p a
-  where
-    injOption = pure . inj
-
 instance Inj p a => Inj p (Data.Semigroup.Last a) where
   inj = pure . inj
 
@@ -402,12 +386,6 @@ instance
   where
     injZipList = fmap inj
 
-instance
-    (DecideOption p ~ Decision_Map, p ~ Option p', Inj p' a) =>
-    InjOption Decision_Map p a
-  where
-    injOption = fmap inj
-
 instance
     (DecideST p ~ Decision_Map, p ~ ST s p', Inj p' a) =>
     InjST Decision_Map p s a
